general/platforms.rst

```diff
@@ -213,9 +213,9 @@ please refer to the file MAINTAINERS_ for contact details for various platforms.
      - No
      - Yes
 
-   * - STMicroelectronics stm32mp1
+   * - `STMicroelectronics stm32mp1 series - <http://www.st.com/stm32mp1>`
```

Comment:
Seems like that on the ST site it's written in upper case (STM32MP1). Overwise ...
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

Reply:
You're right. Thanks. I'll fix.